src/assets/clock.svg

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
This clock icon looks like it has more "white space" on the top than the bottom on my laptop. Could you make the bottom match the top?